src/features/search/components/VnSearchPopover/VnSearchPopover.tsx

```diff
@@ -0,0 +1,18 @@
+import type { FC, PropsWithChildren } from 'react';
+import { memo } from 'react';
+import { IconButton } from 'src/components/IconButton/IconButton';
+import { Popover, PopoverContent, PopoverTrigger } from 'src/components/Popover/Popover';
+
+/** Popover for vn search. */
+const VnSearchPopoverComponent: FC<PropsWithChildren> = ({ children }) => (
+  <Popover>
+    <PopoverTrigger asChild>
+      <IconButton intent="tertiary" name="options" className="shrink-0" />
+    </PopoverTrigger>
+    <PopoverContent align="end" className="w-[560px] px-8 py-7">
```

Comment:
```suggestion
    <PopoverContent align="end" className="w-max-[560px] w-full px-8 py-7">
```
Not sure if it will work, but it should